app/models/manageiq/providers/container_manager.rb

```diff
@@ -56,9 +56,11 @@ class ContainerManager < BaseManager
 
     virtual_column :port_show, :type => :string
 
+    supports     :ad_hoc_metrics
```

Comment:
I thought ad_hoc metrics were gone - This might just be a holdover, or perhaps I'm wrong?  @agrare ?

Reply:
there are a few references to this in ui-services (allPermissions and 2 tests), ui-classic, product_features, and user_roles.

I also thought we had removed it but a few hits via grep and I backed off

Reply:
deleted